### PR TITLE
Prepare for the 1.6.0 release

### PR DESCRIPTION
**Pull request by carmocca**
## What does this PR do?

Preparing engines for takeoff 🚀 

### Does your PR introduce any breaking changes? If yes, please list them.

None

## Before submitting

- [ ] Was this **discussed/approved** via a GitHub issue? (not for typos and docs)
- [x] Did you read the [contributor guideline](https://github.com/PyTorchLightning/pytorch-lightning/blob/master/.github/CONTRIBUTING.md), **Pull Request** section?
- [ ] Did you make sure your **PR does only one thing**, instead of bundling different changes together?
- [ ] Did you make sure to **update the documentation** with your changes? (if necessary)
- [ ] Did you write any **new necessary tests**? (not for typos and docs)
- [ ] Did you verify new and **existing tests pass** locally with your changes?
- [x] Did you list all the **breaking changes** introduced by this pull request?
- [x] Did you **update the [CHANGELOG](https://github.com/PyTorchLightning/pytorch-lightning/blob/master/CHANGELOG.md)**? (not for typos, docs, test updates, or internal minor changes/refactorings)

## PR review

Anyone in the community is welcome to review the PR.
Before you start reviewing make sure you have read [Review guidelines](https://github.com/PyTorchLightning/pytorch-lightning/wiki/Review-guidelines). In short, see the following bullet-list:

- [x] Is this pull request ready for review? (if not, please submit in draft mode)
- [x] Check that all items from **Before submitting** are resolved
- [x] Make sure the title is self-explanatory and the description concisely explains the PR
- [x] Add labels and milestones (and optionally projects) to the PR so it can be classified

cc @borda

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -5,7 +5,90 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
 
 
-## [1.6.0] - 2022-MM-DD
+## [1.7.0] - 2022-MM-DD
```
Comment (carmocca) [inline CHANGELOG.md:8]: Adding this already so PRs for latter milestones can update already without conflicts when we remove spaces and cleanup the 1.6.0 changelog

Comment (Borda) [reply CHANGELOG.md:8]:
```suggestion
## [UnReleased] - 2022-MM-DD
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -5,7 +5,90 @@ All notable changes to this project will be documented in this file.
...
+
+-
+
+
+-
+
+
+## [1.6.0] - 2022-03-29
 
 ### Added
```
Comment (Borda) [inline CHANGELOG.md:93]: I guess that the double blank line shall be also removed before the release...

Comment (carmocca) [reply CHANGELOG.md:93]: Yes. Not doing it here to avoid conflicts to open PRs

**Review by awaelchli — APPROVED** [review]

**Review by akihironitta — APPROVED** [review]
🚀

File: CHANGELOG.md (hunk 1 above)
Comment (kaushikb11) [reply CHANGELOG.md:8]: @carmocca

**Review by kaushikb11 — APPROVED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (carmocca) [reply CHANGELOG.md:8]: For some reason, I cannot commit the suggestion

**Review by Borda — APPROVED** [review]

**Review by justusschock — APPROVED** [review]

**Pull request merged**